projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1187,6 +1190,38 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy {
         return this._visibleWhenCollapsed;
     }
 
+    private _columnPipeArgs: IColumnPipeArgs = { format: DEFAULT_DATE_FORMAT, digitsInfo: DEFAULT_DIGITS_INFO };
+    /**
+     * @remarks
+     * Pass optional parameters for DatePipe and/or DecimalPipe to format the display value for date and numeric columns.
+     * Accepts an `IColumnPipeArgs` object with any of the `format`, `timezone` and `digitsInfo` properties.
+     * For more details see https://angular.io/api/common/DatePipe and https://angular.io/api/common/DecimalPipe
+     * @example
+     * ```typescript
+     * const pipeArgs: IColumnPipeArgs = {
+     *      format: 'longDate',
+     *      timezone: 'UTC',
+     *      digitsInfo: '1.1-2'
+     * }
+     * ```
+     * ```html
+     * <igx-column dataType="date" [pipeArgs]="pipeArgs"></igx-column>
+     * <igx-column dataType="number" [pipeArgs]="pipeArgs"></igx-column>
+     * ```
+     * @memberof IgxColumnComponent
+     */
+    @Input()
+    set pipeArgs(value: IColumnPipeArgs) {
+        this._columnPipeArgs = Object.assign(this._columnPipeArgs, value);
+        this.grid.summaryService.clearSummaryCache();
+        (this.grid as any)._pipeTrigger++;
+        this.grid.summaryService.retriggerRootPipe++;
+        this.grid.notifyChanges(true);
```

Comment:
Just call `notifyChanges()`. In the current state you are needlessly sizing the grid which should not be needed for the pipes.